features/step_definitions/cms/groups_steps.rb

```diff
@@ -40,8 +41,8 @@
   Group.find_by_name(name).should_not be_nil
 end
 
-Then /^I should see the (buyer "[^"]*") belongs to the (buyer group "[^"]*" of provider "[^\"]*")$/ do |account, group|
-  assert account.groups.include?(group)
+Then "I should see the {buyer} {does}( )belong(s) to the {buyer_group_of_provider}" do |account, belongs, group|
```

Comment:
This would require one to look at transformations to understand what how step should be called. Maybe `buyer_group_x_of_provider_y` would give better clue. Not sure.

Reply:
we have multiples of this, I think it can be used as a convention the way you have it already